stream-chat-android-compose/src/main/java/io/getstream/chat/android/compose/ui/components/messageoptions/MessageOptionItemVisibility.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2014-2024 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-chat-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.chat.android.compose.ui.components.messageoptions
```

Comment:
Is the package good or should it be place somewhere else?

Reply:
The package is ok 👍

Reply:
Or maybe we could move it to `io.getstream.chat.android.compose.ui.theme`
@JcMinarro wdyt?

Reply:
As far as this class will be used within the component, it should be "close" to the component. The customer will initialize it in `ChatTheme`, but the real use of this class will be into the Message List Component, so I see the package name is right